model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -123,8 +123,9 @@ def restore_correct_ports(graph: Graph):
             in_port_id = d['in'] if not is_control_flow else 'control_flow_' + str(d['in'])
             to_node_attrs['_in_ports'].update({in_port_id: {'control_flow': is_control_flow}})
         if 'out' in d:
-            num_of_in_nodes = len(Node(graph, u).in_nodes())
-            decremented_number = d['out'] - num_of_in_nodes
+            node = Node(graph, u)
```

Comment:
Please, add comments here why `Const` op is handled specially

Reply:
Done.